metrics/accumulator.go

```diff
@@ -122,15 +122,26 @@ func (a *MetricDataAccumulator) volumeStats(podResource *Resource, s stats.Volum
 }
 
 func (a *MetricDataAccumulator) accumulate(r *Resource, m ...Metrics) {
-	retMetrics := make(Metrics)
+	podName := r.Name
```

Comment:
This should be `resourceName` instead.  Since we call this for any of the nodes, pods, containers, volume resources.